supergenpass.sh

```diff
@@ -3,12 +3,13 @@
 ################################################################################
 
 default_password_length="10"	### Default is 10. It can be overwritten on an individual basis by adding the custom length as a second argument after the domain when invoking the command.
-hashing_algorithm="md5"		### Default is "md5". Alternate value is "sha512".
+default_hashing_algorithm="md5"		### Default is "md5". Alternate value is "sha512". It can overwritten with the third argument.
```

Comment:
Grammar: "It can overwritten" => "It can _be_ overwritten"

Reply:
Done